### ISSUE TITLE
What exactly does `tf.signal.fft` compute?

### ISSUE THREAD

**Issue by sharvil** [tensorflow/tensorflow#40709]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/signal/fft

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/signal/fft

## Description of issue (what needs changing):
The description is simply "Fast Fourier transform." which isn't fully-specified. What is the exact function computed? Is there a normalization term of 1/sqrt(N) or 1/N? Or is the normalization constant entirely in the inverse FFT (which has equally underspecified documentation)?

### Clear description
A mathematical formula that specifies what `tf.signal.fft` implements would be nice. Likewise with the other FFT methods, the inverse FFT methods, and STFT methods.

**Comment by sharvil** [comment]
https://pytorch.org/docs/master/generated/torch.fft.html can serve as a template for the kind of information that a developer would find useful.

**Comment by lamberta** [comment]
This should be the file to edit the docstring: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/base_api/api_def_FFT.pbtxt
Pull requests welcome 😀

**Comment by jvishnuvardhan** [comment]
@sharvil Are you interested in raising a PR to update the docs? Thanks!

**Comment by sharvil** [comment]
@jvishnuvardhan, no, I won't be submitting a PR to update these docs.

**Comment by Patryk999** [comment]
I would like to work on this issue

**Comment by jvishnuvardhan** [comment]
@Patryk999 Please feel free to open a PR to update the doc and mention this issue number in that PR. Thanks!

**Comment by sharvil** [comment]
.

**Comment by MarkDaoust** [comment]
Here's  quick test:

https://colab.research.google.com/drive/1pm-CJXaeuti1OT-O1vloH_BtjoM3by8H?authuser=1#scrollTo=9hW4DhoNJjcy

The result is pretty clearly N-times the amplitude (rfft only returns the first half of the amplitudes).

I'll see if I can patch those docs quick.
